api/src/main/java/org/jboss/forge/roaster/Roaster.java

```diff
@@ -194,26 +262,36 @@ private Roaster()
          {
             return (T) unit.getGoverningType();
          }
-         else if (unit != null)
-         {
-            throw new ParserException("Source does not represent a [" + type.getSimpleName() + "], instead was ["
-                     + unit.getGoverningType().getClass().getSimpleName() + "] - Cannot convert.");
-         }
+         throw new ParserException("Source does not represent a [" + type.getSimpleName() + "], instead was ["
+                  + unit.getGoverningType().getClass().getSimpleName() + "] - Cannot convert.");
       }
-      throw new ParserException("Cannot find JavaParser capable of parsing the requested data");
```

Comment:
This is still required for cases when roaster-jdt is not in classpath

Reply:
Valid point. Will change this